frontend/pages/ManageControlsPage/MacOSSetup/cards/BootstrapPackage/components/BootstrapPackageTable/BootstrapPackageTable.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { useQuery } from "react-query";
+
+import { IBootstrapPackageAggregate } from "interfaces/mdm";
+import mdmAPI from "services/entities/mdm";
+
+import DataError from "components/DataError";
+import TableContainer from "components/TableContainer";
+import EmptyTable from "components/EmptyTable";
+
+import {
+  generateTableData,
+  generateTableHeaders,
+} from "./BootstrapPackageTableConfig";
+
+const baseClass = "bootstrap-package-table";
+
+interface IBootstrapPackageTableProps {
+  currentTeamId: number;
+}
+
+const DEFAULT_SORT_HEADER = "hosts";
+const DEFAULT_SORT_DIRECTION = "asc";
+
+const BootstrapPackageTable = ({
+  currentTeamId,
+}: IBootstrapPackageTableProps) => {
+  const { data: bootstrapPackageAggregatem, isLoading, isError } = useQuery<
+    IBootstrapPackageAggregate,
+    Error,
+    IBootstrapPackageAggregate
+  >(
+    ["bootstrap-package-summary", currentTeamId],
+    () => mdmAPI.getBootstrapPackageAggregate(currentTeamId),
+    {
+      retry: false,
+      refetchOnWindowFocus: false,
+    }
+  );
+
+  const tableHeaders = generateTableHeaders();
+  const tableData = generateTableData(
+    bootstrapPackageAggregatem,
+    currentTeamId
+  );
```

Comment:
Since I'm on the general topic of tables, I think the `generateTableData` pattern might be better if we perform it as part of the `QueryFunction` of the `useQuery` hook so that the results of the transform get memoized as part of the cache. Not for this PR so I'll add these ideas to the frontend sync agenda.